scipy/stats/tests/common_tests.py

```diff
@@ -113,7 +113,7 @@ def check_entropy(distfn, arg, msg):
 
 def check_private_entropy(distfn, args, superclass):
     # compare a generic _entropy with the distribution-specific implementation
-    npt.assert_allclose(distfn._entropy(*args),
+    npt.assert_allclose(distfn.entropy(*args),
```

Comment:
I don't intend to actually make these changes to the test. I just wanted to see what needed to be done to make tests pass.